src/LanguageServer/ExternalAccess/Copilot/LspServices/AbstractCopilotLspService.cs

```diff
@@ -0,0 +1,9 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.CodeAnalysis.LanguageServer.ExternalAccess.Copilot;
+
+internal class AbstractCopilotLspService : ILspService
```

Comment:
abstract type that is not actually abstract?

also, what is this for?  Can it be documented?